tutorials/advanced_features/Simulating with laser noises.ipynb

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
<ul><li>"coming from the laser itself, encapsulating effects like the thermal motion of the atoms giving rise to a doppler shift on the laser frequency. "</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>